crates/nu-command/src/default_context.rs

```diff
@@ -172,7 +172,6 @@ pub fn add_shell_command_context(mut engine_state: EngineState) -> EngineState {
         ))]
         bind_command! { Ps };
 
-        #[cfg(feature = "which-support")]
         bind_command! { Which };
```

Comment:
Could this go into another one of the bind_command! calls now instead of being here by itself? Perhaps with all the system commands, or at least the miscellaneous commands?